It looks that original Broadcom CS00011797870 patch unnecessarily moved MFREE call inside
the "if" clause. In result, after many network scans, we observe that following "wl"
call on that radio reports "No memory" error.

--- a/bcmdrivers/broadcom/net/wl/impl69/main/src/wl/sys/wlc_dfs.c
+++ b/bcmdrivers/broadcom/net/wl/impl69/main/src/wl/sys/wlc_dfs.c
@@ -536,8 +536,8 @@ wlc_dfs_cac_state_change(wlc_dfs_info_t
 			(event_len - sizeof(wlc_cac_event_t) + sizeof(wl_dfs_status_all_t)));
 		wlc_bss_mac_event(wlc, cfg, WLC_E_CAC_STATE_CHANGE, NULL, 0, 0, 0,
 			cac_event, event_len);
-		MFREE(wlc->osh, cac_event, event_len);
 	}
+	MFREE(wlc->osh, cac_event, event_len);
 }
 
 /* called when observing toggle between EU to/from non-EU country */
